src/lib/storage/table.cpp

```diff
@@ -165,7 +165,7 @@ void Table::append_chunk(const Segments& segments, const std::optional<Polymorph
   std::shared_ptr<MvccData> mvcc_data;
 
   if (_use_mvcc == UseMvcc::Yes) {
-    mvcc_data = std::make_shared<MvccData>(chunk_size);
+    mvcc_data = std::make_shared<MvccData>(chunk_size, CommitID{0});
```

Comment:
would this ever be anything else than zero? If not remove the second constructor argument?

Reply:
Rather make it a bit more explicit. Added a comment.